test/clt-tests/sharding/replication/replication-test.rec

```diff
@@ -0,0 +1,302 @@
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/stop-searchd –––
+––– input –––
+mysql -h0 -P3306 -e "INSERT INTO replication:testrt VALUES(0,'NEW','TABLE',4);"; echo $?
+––– output –––
+0
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+sleep 5;mysql -h0 -P2306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P3306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P4306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+mysql -h0 -P5306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: NEW
+content: TABLE
+gid: 4
+*************************** 4. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
```

Comment:
You repeat this select command many times, and only node port was changed. Maybe you should expose this code into the `block` and use it instead?

Reply:
This looks ok to me. Creating a block for each select/resultset when we have to call it a few times seems excessive. A block should be something less specific, e.g. a global block for starting `searchd`.